dronegen/tests.go

```diff
@@ -75,9 +75,18 @@ func testCodePipeline() pipeline {
 	}
 	p.Trigger = triggerPullRequest
 	p.Workspace = workspace{Path: "/go"}
-	p.Volumes = dockerVolumes(volumeTmpfs, volumeTmpDind, volumeTmpIntegration)
+	p.Volumes = dockerVolumes(
+		volumeTmpfs,
+		volumeTmpDind,
+		volumeTmpIntegration,
+		volumeDockerTmpfs,
+	)
 	p.Services = []service{
-		dockerService(volumeRefTmpfs, volumeRefTmpDind),
+		dockerService(
+			volumeRefTmpfs,
+			volumeRefDockerTmpfs,
+			volumeRefTmpDind,
+		),
```

Comment:
hmm, does this need a `volumeTmpIntegration` ref?

Reply:
I think this was originally done to give each pipeline step an isolated `/tmp` that wasn't on overlay storage (because the tests make extensive use of it)

It may not be relevant any more.